### PR TITLE
simply "easier to fork" feature as "open source"

### PR DESCRIPTION
**Pull request by pdurbin**
As discussed with @holmesworcester in #general, I find the "Easier to fork" feature a bit confusing and hard to follow, even after reading [a similar argument](https://github.com/TryQuiet/quiet/wiki/Quiet-FAQ#slack-and-discord) in the FAQ.

In this pull request I'm suggesting what I think is a simplification of the argument: no vendor lock-in because Quiet is open source. I also added a line about interoperability to try to remain true to the original intent but I don't think it's strictly necessary.

## Old: Easier to fork

Screen Shot 2022-12-17 at 4 57 34 PM

## New: Open source

Screen Shot 2022-12-17 at 4 57 39 PM

### CODE REVIEW

**Review by holmesworcester — APPROVED** [review]
I may tweak this some more but it's an improvement over what's there!

**Comment by pdurbin** [comment]
Sounds good! Absolutely tweak away!
